### ISSUE TITLE
Enable webGME to run under systemd control

### ISSUE THREAD

**Issue by phreed** [webgme/webgme#1205]
The obvious approach is to add the following to <gme-app>/config/config.default.js
```
...
var systemdSocket = require("systemd-socket");
...
config.server.port = systemdSocket() || 3000;
...
```
The difficulty is that webgme/config/validator.js has an overly strict constraint, i.e. more restrictive than the [method it is protecting](https://nodejs.org/api/http.html#http_server_listen_handle_callback).
Which is [called from here]( https://github.com/webgme/webgme/blob/master/src/server/standalone.js#L216).
At issue is the fact that systemdSocket(0) does not return a number but rather an object of the form '{fd: 3}'.
The change would be in https://github.com/webgme/webgme/blob/master/config/validator.js#L194
Relaxing the constraint to allow an object as well as a number.

**Comment by pmeijer** [comment]
This can be fixed, just a slight difference is needed since we use the port in other places too (where it has to be a number). 
So we would still require the port to be given, but add an optional parameter `config.server.systemd` or do you have a better name?

**Comment by pmeijer** [comment]
actually `config.server.handle` is better.

**Comment by pmeijer** [comment]
Closed by #1206
